source/nodejs/adaptivecards-site/themes/adaptivecards/layout/index.ejs

```diff
@@ -18,11 +18,11 @@
 			<div class="accent-paragraph"><%- site.data.home.en.heading_paragraph %></div>
```

Comment:
@paulcam206 @KesemZ This bug also called out several links within the UHF portion of our site. Should I fix those in our hard copy, transfer the bug to the UHF team, or both?